src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -306,22 +251,18 @@ BinaryCompactObject::BinaryCompactObject(
     std::array<double, 2> initial_size_map_values,
     std::array<double, 2> initial_size_map_velocities,
     std::array<double, 2> initial_size_map_accelerations, Object object_A,
-    Object object_B, double radius_enveloping_cube, double outer_radius_domain,
+    Object object_B, double envelope_radius, double outer_radius,
     const typename InitialRefinement::type& initial_refinement,
     const typename InitialGridPoints::type& initial_number_of_grid_points,
     const bool use_equiangular_map, bool use_projective_map,
-    double frustum_sphericity,
-    const std::optional<double>& radius_enveloping_sphere,
     CoordinateMaps::Distribution radial_distribution_outer_shell,
     std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>
         outer_boundary_condition,
     const Options::Context& context)
     : BinaryCompactObject(std::move(object_A), std::move(object_B),
-                          radius_enveloping_cube, outer_radius_domain,
-                          initial_refinement, initial_number_of_grid_points,
-                          use_equiangular_map, use_projective_map,
-                          frustum_sphericity, radius_enveloping_sphere,
-                          radial_distribution_outer_shell,
+                          envelope_radius, outer_radius, initial_refinement,
+                          initial_number_of_grid_points, use_equiangular_map,
```

Comment:
I think keeping `use_equiangular_map` for now this good (and changing it would be out of the scope of this PR) but as a note to myself I'll keep in mind that once we have `use_equiangular_map == true` working consistently better than the `false` case maybe it will be time to also default this option. (But I don't think we're there yet as @geoffrey4444 is observing constraint error growth in the outer layer when this option is turned on)